### PR TITLE
Responsive fixes + disable SSR + optional chaining

### PR DESCRIPTION
**Pull request by palmenhq**
I feel like we're not getting much from the SSR compared to what we have to put in, thoughts?

### CODE REVIEW

File: src/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx [hunk 1]
```diff
@@ -223,7 +223,7 @@ export const DiscountCodeModal: React.FC<Props> = ({
                     <TranslationsConsumer textKey="SIDEBAR_ADD_DISCOUNT_FINEPRINT">
                       {(t) => t}
                     </TranslationsConsumer>{' '}
-                    <TermsLink href="">
+                    <TermsLink href="" target="_blank">
```
Comment (hedvigoscar) [inline src/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx]:
```suggestion
                    <TermsLink href="" target="_blank" rel="noopener noreferrer">
```

File: src/pages/OfferNew/Compare/CompareTable.tsx [hunk 2]
```diff
@@ -358,31 +387,53 @@ export const CompareTable = (props: Props) => {
...
               }}
             >
-              {company.name}
-            </DropdownRow>
-          ))}
-        </Dropdown>
+              {currentCompany !== null && !dropdownIsVisible
+                ? currentCompany.name
+                : 'Välj bolag'}
+              {/* TODO: text key? ☝️ */}
```
Comment (gustafguner) [inline src/pages/OfferNew/Compare/CompareTable.tsx]: I added a text key for it, you can use `COMPARE_TABLE_DROPDOWN_LABEL` :)

File: src/pages/OfferNew/Compare/CompareTable.tsx [hunk 3]
```diff
@@ -358,31 +387,53 @@ export const CompareTable = (props: Props) => {
...
+        {isMobile && (
+          <MobileDropdown
+            onChange={(e) =>
+              setCurrentCompany(
+                otherCompanies.find(({ name }) => e.target.value === name) ||
+                  null,
+              )
+            }
+            value={currentCompany?.name}
+            defaultValue="Välj bolag" // TODO text key?
```
Comment (gustafguner) [inline src/pages/OfferNew/Compare/CompareTable.tsx]: Same text key here and below

File: src/pages/OfferNew/Perils/PerilSwiper/index.tsx [hunk 4]
```diff
@@ -59,12 +51,23 @@ const Dot = styled.div<DotProps>`
...
-    [],
-  ) as Peril[][]
+    [] as Array<typeof array>,
+  )
+
+const PAGE_GUTTER = 32 * 2
+const PERIL_SIZE = 138
+const PERIL_MARGIN = 16
+const PERIL_ROWS = 2
+const getNumberOfPerilsPerSlide = (windowWidth: number) =>
```
Comment (gustafguner) [inline src/pages/OfferNew/Perils/PerilSwiper/index.tsx]: Really nice!

File: src/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx (hunk 1 above)
Comment (palmenhq) [reply src/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx]: yes, nice catch. thanks!

**Pull request merged**